Getting-Started-with-Akka-http-signature.md

```diff
@@ -0,0 +1,232 @@
+***
+
```

Comment:
All of this section is about using Akka-http-signature as a client with an existing server. One implementation of the Http-signature protocol is rww-play (though it too could be using the akka-http-signature library as a client, and will in a future release)